dev/io.openliberty.mcp.internal/resources/io/openliberty/mcp/internal/CWMCM.nlsprops

```diff
@@ -0,0 +1,202 @@
+CWMCM0003E.missing.tool.argument.name.useraction=Either add a name to the @ToolArg annotation, or to add the -parameters compiler option to use the parameter name.
+
+# for server
+# do not translate MCP
+# {0} = The list of qualified names of the tool
+CWMCM0004E.duplicate.tools=CWMCM0004E: Multiple MCP tool have the same name: {0}.
+CWMCM0004E.duplicate.tools.explanation=One or more MCP Tool names occured multiples times.The server expects one name per tool for uniqueness.
+CWMCM0004E.duplicate.tools.useraction=Only provide a unique name for each MCP Tool.
+
+# for server
+# do not translate MCP
+# {0} = The fully qualified method name
+# {1} = The parameter type simple name
+CWMCM0005E.duplicate.special.arguments=CWMCM0005E: The {0} MCP Tool has more than one parameter with type {1}. There may only be one {1} parameter for each Tool method.
+CWMCM0005E.duplicate.special.arguments.explanation=One or more special arguments occured multiples times, only one is expected.
+CWMCM0005E.duplicate.special.arguments.useraction=Remove the extra instances.
+
+# for server
+# {0} = The fully qualified method name
+# {1} = The parameter type simple name
+CWMCM0006E.invalid.arguments=CWMCM0006E: The {0} MCP Tool has a parameter of type {1} which is not a recognised special argument type and does not have a `@ToolArg` annotation.
+CWMCM0006E.invalid.arguments.explanation=One or more parameters were not annotated as tool arguments and are not recognised special argument types.
+CWMCM0006E.invalid.arguments.useraction=Please check if you have the correct class imported for your argument, or you meant to include @ToolArg.
+
+# for client
+# {0} = Request id
+CWMCM0007E.invalid.request.params=CWMCM0007E: A request with ID {0} is already in progress.
+CWMCM0007E.invalid.request.params.explanation= A request with the same id is still in progress. Requests from the same client must have unique IDs.
+CWMCM0007E.invalid.request.params.useraction=Use a unique ID for every request
+
+# for client
+# do not translate GET, POST
+CWMCM0008I.get.disallowed=CWMCM0008I: GET requests are not supported.
+CWMCM0008I.get.disallowed.explanation=A GET request was sent but this functionality is not supported.
+CWMCM0008I.get.disallowed.useraction=Use a supported HTTP request i.e POST
+
+# for client
+# do not translate JSON-RPC, JSON
+CWMCM0009E.jsonrpc.parse.error=CWMCM0009E: The JSON-RPC request is not valid JSON.
+CWMCM0009E.jsonrpc.parse.error.explanation=The JSON-RPC request could not be parsed by the server due to invalid JSON being parsed. 
+CWMCM0009E.jsonrpc.parse.error.useraction=Correct the JSON-RPC request message.
+
+# for client
+# do not translate JSON-RPC, JSON
+CWMCM0010E.jsonrpc.invalid.request=CWMCM0010E: The JSON-RPC request was invalid.
+CWMCM0010E.jsonrpc.invalid.request.explanation=The JSON recieved is not a valid JSON-RPC request object.
+CWMCM0010E.jsonrpc.invalid.request.useraction=Fix the issues in the request that have been highlighted.
+
+# for client
```

Comment:
Can you do a find and replace with `# for client` and replace it with "Used as part of a response to a client".

It's important when we review these messages that we know where they're going. I'm tempted to separate them into a different file and number range to make it harder to mix them up with messages that are logged to the server. I'll think a bit more about that.